aiops/opni-opensearch-update-service/Readme.md

```diff
@@ -0,0 +1,5 @@
+## Testing the Opensearch Updating Service
+Simply run 
+```
+python3 test_opensearch_update.py
```

Comment:
Tests should be run using `pytest` - Also please rename to README.md if you want to keep this file